EventListener/LoadORMEmbeddedMetadataSubscriber.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * This file is part of the YAP package
+ *
+ * (c) Sébastien COTON <sebastien.coton@gmail.com>
+ * (c) Ouat's UP
+ * (c) SAS Chaucodel
+ *
+ * L'usage et la diffusion de ce fichier ainsi que de l'ensemble des fichiers
+ * liés au présent projet doit faire l'objet d'un accord express
+ * de l'ensemble des parties citées dans le (c) ci-dessus
+ *
+ */
```

Comment:
Do you want to keep this comment here?

Reply:
damn sorry, no, it can be dropped  :))))

Reply:
Maybe better if i should prepare another PR ? (it's almost my first contrib) :))